templates/themes/1/template_1_article.amp.html

```diff
@@ -0,0 +1,109 @@
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!doctype html>
+{{#article}}
+<html ⚡>
+{{> ../utils/amp-page-head.snip.html}}
+<body>
+  {{> ../components/navbar/navbar.snip.html}}
+
+  {{#fullpage-hero}}
+    <!-- Start Fullpage Hero -->
+    <figure class="ampstart-image-fullpage-hero m0 relative">
+      <amp-img width="404" height="720" layout="responsive" src="{{{narrow-img}}}" media="(max-width: 415px)"></amp-img>
+      <amp-img height="720" layout="fixed-height" src="{{{wide-img}}}" media="(min-width: 416px)"></amp-img>
+      <figcaption class="absolute bottom-0 right-0 left-0 center m3">
+        <header class="p3">
+          <h1 class="ampstart-fullpage-hero-heading h2 mb1">
+            <span class="ampstart-fullpage-hero-heading-text">
+              {{{heading}}}
+            </span>
+          </h1>
+          {{#credit}}
+            <span class="ampstart-image-credit h3 bold">
+              {{credit-prefix}} {{#author}}<a href="{{{url}}}" role="author" class="text-decoration-none">{{text}}</a>{{/author}}{{#credit-suffix}}, {{.}}{{/credit-suffix}}
+            </span>
+          {{/credit}}
+        </header>
+        <footer>
+          {{#readmore}}
+            <a class="ampstart-readmore" href="#{{content-id}}">{{text}}</a>
+          {{/readmore}}
+        </footer>
+      </figcaption>
+    </figure>
+    <!-- End Fullpage Hero -->
+  {{/fullpage-hero}}
+
```

Comment:
I guess my comment vanished here. I dont see a difference between this and the one in the component, what is different here ?